### ISSUE TITLE
Add support for Special Builds of NSIS.

### ISSUE THREAD

**Issue by 3D1T0R** [PortableApps/Launcher#7]
As discussed on the "[Chrome Portable command line length limit](https://portableapps.com/node/56728)" thread, there are some "Special Builds" of NSIS released along with the default build which allow for things the default build can't do. In particular, interest was expressed in using the "Large Strings" build for Google Chrome Portable so that users could pass a larger number of Chrome's parameters to it without NSIS truncating the execution string.

I have an implementation which I intend to submit for review before it gets into the main code base, but first I would like to have a few questions answered:
- [ ]  What version number are we intending to release next? 2.2.2 or 3.0?
- [ ]  Should both of the "Special Builds" be included? or should we skip the "Advanced Logging" build and only include the "Large Strings" build?
- [ ] What should the build(s) be called?
    I'm leaning toward "LargeStrings" and "AdvancedLogging" as these match what they're called on NSIS' [Special Builds](http://nsis.sourceforge.net/Special_Builds) page

Note: I've put the questions in a task list so I can check each one off when I feel we have a satisfactory answer for it. Once they're all checked off, I'll upload my implementation.

**Comment by JohnTHaller** [comment]
One note: Chrome Portable does not currently use PAL. It uses a custom launcher.

**Comment by 3D1T0R** [comment]
True, but I was under the impression we wanted to switch it to PAL at some point.
Plus, this may also end up being useful for other apps.
